x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/analysis/VerifierTests.java

```diff
@@ -111,8 +111,8 @@ public void testArrayFunctionsUnsupported() {
 
     // Some functions fail with "Unknown" message at the parse stage
     public void testFunctionParsingUnknown() {
-        assertEquals("1:15: Unknown function [matchLite]",
-                error("process where matchLite(?'.*?net1\\s+localgroup\\s+.*?', command_line)"));
+        assertEquals("1:15: Unknown function [matchLite], did you mean any of [matchLite, match]?",
```

Comment:
Any idea why I'm getting an error here at all?

Reply:
This being a test unit, it would have been easier to debug in an IDE.